c/cbot.txt

```diff
@@ -0,0 +1,6 @@
+extern void object::New()
+{
+
+message("Hello world");
```

Comment:
```suggestion
message("Hello World");
```

Reply:
No, looks almost like the javascript alert command

ach, typo right

Reply:
@kompowiec All the examples have to display the message "Hello World" in title case.

Reply:
yeah i noticed there was a typo, next time i will remember.